README.md

```diff
@@ -60,12 +60,14 @@ OSM is under active development and is **NOT** ready for production workloads.
 
 ### SMI Specification support
 
-|   Specification Component    |         Supported Release          |          Comments          |
-| :---------------------------- | :--------------------------------: |  :--------------------------------: |
-| Traffic Access Control  |  [v1alpha3](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-access/v1alpha3/traffic-access.md)  | |
-| Traffic Specs  |  [v1alpha4](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-specs/v1alpha4/traffic-specs.md)  | |
-| Traffic Split  |  [v1alpha2](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-split/v1alpha2/traffic-split.md) | |
-| Traffic Metrics  | [v1alpha1](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-metrics/v1alpha1/traffic-metrics.md) | 🚧 **In Progress** [#379](https://github.com/openservicemesh/osm/issues/379) 🚧 |
+|   Kind    | SMI Resource |         Supported Release          |          Comments          |
```

Comment:
Since we will not be supporting all SMI API versions in all versions of OSM, it would be good to add a column for `OSM version`, and change the table to something as follows:
```
| SMI Kind | SMI API version | SMI Resource | OSM version | Comments |
```

Reply:
Do you mean that column should represent the versions of OSM that also support this specific CRD version? e.g. "traffictargets.access.smi-spec.io/v1alpha3 is supported by OSM `>=v0.7.0`"

If you mean the table should show for each version of OSM the SMI versions it supports, then I'm not sure how best to show that. Would it be sufficient to read the table as "these are the SMI versions supported by this version of OSM" in that case? Then if the supported SMI versions change for a future version of OSM, the change will be made in the readme then. We wouldn't backport updates to this table to all past versions of OSM.

If we want a global map for all versions of OSM, then it seems like that should live outside the rest of the versioned source if we're concerned people may be looking at an out-of-date version of the table.

Reply:
That sounds reasonable to me. Let's keep it simple as you have already and revisit this when we end up needing to document support for multiple versions.